var/spack/repos/builtin/packages/openspeedshop-utils/package.py

```diff
@@ -0,0 +1,345 @@
+
+    depends_on('cbtf-krell@develop+crayfe', when='@develop+crayfe')
+    depends_on('cbtf-krell@1.9.1.0+crayfe', when='@2.3.1.3+crayfe')
+
+    depends_on('cbtf-krell@develop+cti', when='@develop+cti')
+    depends_on('cbtf-krell@1.9.1.0+cti', when='@2.3.1.3+cti')
+
+    depends_on('cbtf-krell@develop+mpich', when='@develop+mpich')
+    depends_on('cbtf-krell@1.9.1.0+mpich', when='@2.3.1.3+mpich')
+
+    depends_on('cbtf-krell@develop+mpich2', when='@develop+mpich2')
+    depends_on('cbtf-krell@1.9.1.0+mpich2', when='@2.3.1.3+mpich2')
+
+    depends_on('cbtf-krell@develop+mpt', when='@develop+mpt')
+    depends_on('cbtf-krell@1.9.1.0+mpt', when='@2.3.1.3+mpt')
+
+    depends_on('cbtf-krell@develop+mvapich', when='@develop+mvapich')
+    depends_on('cbtf-krell@1.9.1.0+mvapich', when='@2.3.1.3+mvapich')
+
+    depends_on('cbtf-krell@develop+mvapich2', when='@develop+mvapich2')
+    depends_on('cbtf-krell@1.9.1.0+mvapich2', when='@2.3.1.3+mvapich2')
+
+    depends_on('cbtf-krell@develop+openmpi', when='@develop+openmpi')
+    depends_on('cbtf-krell@1.9.1.0+openmpi', when='@2.3.1.3+openmpi')
+
+    depends_on("cbtf-argonavis@develop", when='@develop+cuda')
+    depends_on("cbtf-argonavis@1.9.1.0", when='@2.3.1.3+cuda')
+
+    # For MRNet
+    depends_on("mrnet@5.0.1-3:+cti", when='@develop+cti')
+    depends_on("mrnet@5.0.1-3:+lwthreads", when='@develop')
+
+    depends_on("mrnet@5.0.1-3:+cti", when='@2.3.1.3+cti')
+    depends_on("mrnet@5.0.1-3:+lwthreads", when='@2.3.1.3')
+
+    parallel = False
+
+    build_directory = 'build_openspeedshop'
+
+    def set_CrayLoginNode_cmakeOptions(self, spec, cmakeOptions):
+        # Appends to cmakeOptions the options that will enable the appropriate
+        # Cray login node libraries
+
+        CrayLoginNodeOptions = []
+        rt_platform = "cray"
+
+        # How do we get the compute node (CNL) cbtf package install
+        # directory path?
+        # spec['cbtf'].prefix is the login node value for this build, as
+        # we only get here when building the login node components and
+        # that is all that is known to spack.
+        be_ck = spack.store.db.query_one('cbtf-krell arch=cray-CNL-haswell')
+
+        # Equivalent to install-tool cmake arg:
+        # '-DCBTF_KRELL_CN_RUNTIME_DIR=%s'
+        #               % <base dir>/cbtf_v2.3.1.release/compute)
+        CrayLoginNodeOptions.append('-DCBTF_KRELL_CN_RUNTIME_DIR=%s'
+                                    % be_ck.prefix)
+        CrayLoginNodeOptions.append('-DRUNTIME_PLATFORM=%s'
+                                    % rt_platform)
+
+        cmakeOptions.extend(CrayLoginNodeOptions)
+
+    def cmake_args(self):
+
+        spec = self.spec
+
+        compile_flags = "-O2 -g"
+
+        cmake_args = []
+
+        # Indicate building cbtf vers (transfer rawdata files)
+        instrumentor_setting = "cbtf"
+
+        if spec.satisfies('+runtime'):
+            # Appends base options to cmake_args
```

Comment:
Not sure what this comment adds (e.g. it's the same in both branches of the if statement)

Reply:
I will remove.